Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -117,15 +147,19 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         let (project, graph) = try subject.map(tuistPath: tuistPath,
                                                sourceRootPath: sourceRootPath,
                                                xcodeProjPath: xcodeProjPath,
+                                               setupPath: nil,
+                                               configPath: nil,
                                                manifests: manifestPaths,
                                                helpers: helperPaths,
                                                templates: templates,
                                                projectDescriptionPath: projectDescriptionPath)
 
         // Then
         let targetNodes = graph.targets.values.flatMap { targets in targets.compactMap { $0 } }.sorted(by: { $0.target.name < $1.target.name })
+
+        // expecting `targetNodes == [TemporaryDirectory.XXXMainifest]`
         XCTAssertEqual(targetNodes.count, 1)
-        XCTAssertEqual(targetNodes.first?.dependencies, [])
+        XCTAssertEqual(targetNodes.first?.dependencies.sorted(by: { $0.name < $1.name }), [])
```

Comment:
is it necessary to perform the `sort`? You can even use `XCTAssertEmpty` which would be a better fit here,  I think.